package.json

```diff
@@ -82,6 +83,7 @@
     "@typescript-eslint/parser": "^3.6.0",
     "conventional-changelog-angular": "^5.0.10",
     "conventional-recommended-bump": "^6.0.9",
+    "cross-env": "^7.0.3",
```

Comment:
If the above scripts do not work, run `npm un cross-env` to uninstall `cross-env` and automatically update `package-lock.json`